python/tchannel/tornado/dispatch.py

```diff
@@ -54,10 +52,10 @@ def __init__(self):
         self.endpoints = {}
 
     @tornado.gen.coroutine
-    def _call_endpoint(self, endpoint, request, response):
+    def _call_endpoint(self, endpoint, request, response, proxy):
         try:
             yield tornado.gen.maybe_future(
-                endpoint.handler(request, response, endpoint.opts))
+                endpoint.handler(request, response, proxy))
         except:
             # TODO send application error
             pass
```

Comment:
The catch all hides away all errors, including critical errors that should not be caught. In this specific case, the widest we should go is `Exception`, and we should log details even if we're currently ignoring the actual exception.

Also see: http://stackoverflow.com/questions/18982610/difference-between-except-and-except-exception-as-e-in-python